### DISCUSSION TITLE
TuringCourses/major/machine_learning/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#19] [General]
# TuringCourses/major/machine_learning/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/machine_learning/

**Comment by floatshadow** [comment]
ML 课程看老师，19级蔡老师会有课后作业

**Comment by MTDickens** [comment]
可以通过 Coursera 上林轩田的机器学习课程，以及 [Deep Learning for Computer Vision](https://web.eecs.umich.edu/~justincj/teaching/eecs498/FA2020/) 两门课，将这门课的知识点（基本）完全覆盖。

**Comment by 626lt** [comment]
机器学习作为人工智能最重要的一部分无疑是值得好好学习的，然而就这门课而言所教授的很多都是二十年前所流行的，比如决策树，支持向量机等等。其中的思想固然重要，但从实际出发无论是科研还是工作都不太会用到这里的东西。
老师对教学质量的不负责也导致这门课整体下来体验极差，到最后考试近乎变成一种如文科般的记忆式了。同时西瓜书的内容含量其实并不小，每一章都是一系列的算法组成的，这门课上起来的感觉十分割裂，根本没有对算法和原理的深入解读。有志于从事这方面科研的同学不要把希望寄于这一门课上，哪怕是当作一门入门的基础课在zz老师的手里也是不合格的。
而且这门课的考核也是有诸多槽点，作业是由生成式人工智能生成的，最后几次课的课件也是，存在大量未经验算的数值错误。不建议非必修的同学来选修，学学网课效率更高效果更好。

**Comment by ken0yuan** [comment]
赵俊博老师的上课比较一般，他自己也承认不想上课，不过助教gxj非常nice，作业晚交理由得当不扣分，期末ddl在考完试后，作业是西瓜书上的，还有编程作业，是实验课和理论课的综合，善用ai，期末面向复习课和复习课上的历年卷能完成大部分的复习，难点不会复习到，虽然zjb有两次小测，有些地方有点深，但似乎对总评的影响不大，是算在平时分上的，平时没有签到，小测有提醒，入股不亏，平时还有课上的实验课，个人觉得非常鸡肋，就是纯抄，没时间理解，后续也会发全部的代码参考，总的来说是可以全翘的。
